install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -225,3 +221,8 @@ global:
   # Use the Mesh Control Protocol (MCP) for configuring Mixer and
   # Pilot. Requires galley (`--set galley.enabled=true`).
   useMCP: false
+
+  sds:
+    enabled: false
+    sdsUdsPath: ""
```

Comment:
Suggest to be consistent with previous attribute, i.e., just call it udsPath and enableTokenMount

Reply:
renamed.